torch/testing/_internal/common_utils.py

```diff
@@ -1129,7 +1129,7 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
         return r
 
 # Run PyTorch tests with TorchDynamo
-TestEnvironment.def_flag("TEST_WITH_TORCHINDUCTOR", env_var="PYTORCH_TEST_WITH_INDUCTOR")
+TestEnvironment.def_flag("TEST_WITH_TORCHINDUCTOR", env_var="TEST_WITH_INDUCTOR")
```

Comment:
I don't think this is enough, because of https://github.com/pytorch/pytorch/blob/main/test/run_test.py#L1665
And regardless, all similar variables starts with `PYTORCH_`
So the message needs to be changed,  not the var name.

Reply:
Ok, the original message seem to imply that the var name should be changed but I understand what you mean.